src/Symfony/Component/PropertyInfo/PropertyInfoExtractor.php

```diff
@@ -51,6 +53,14 @@ public function getLongDescription(string $class, string $property, array $conte
         return $this->extract($this->descriptionExtractors, 'getLongDescription', [$class, $property, $context]);
     }
 
+    public function getType(string $class, string $property, array $context = []): ?Type
+    {
+        return $this->extract($this->typeExtractors, 'getType', [$class, $property, $context]);
+    }
+
+    /**
+     * @deprecated since Symfony 7.1, use "getType" instead.
```

Comment:
shouldn't the method trigger a runtime deprecation also? (same in all deprecated methods)